src/view.js

```diff
@@ -96,7 +96,7 @@ const View = Backbone.View.extend({
     Backbone.View.prototype.setElement.apply(this, arguments);
 
     if (hasEl) {
-      this._isRendered = !!this.$el.length;
+      this._isRendered = this.Dom.hasContents(this.el);
```

Comment:
Yeah I was on the fence about it and could go the other way.  But I'm leaning towards it all being on the api at the moment xref: https://github.com/marionettejs/backbone.marionette/issues/3398 

I suppose it comes down to if the DomApi and "utils" like `isNodeAttached` are useful directly outside of the internal uses or if we should let users do it on their own.